capa/features/extractors/cape/extractor.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (C) 2020 Mandiant, Inc. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at: [package root]/LICENSE.txt
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+#  is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+
+import logging
+from typing import Dict, Tuple, Iterator
+
+import capa.features.extractors.cape.global_
+import capa.features.extractors.cape.process
+import capa.features.extractors.cape.file
+import capa.features.extractors.cape.thread
+from capa.features.common import Feature
+from capa.features.address import Address
+from capa.features.extractors.base_extractor import ProcessHandle, ThreadHandle, DynamicExtractor
+
+logger = logging.getLogger(__name__)
+
+
+class CapeExtractor(DynamicExtractor):
+    def __init__(self, static: Dict, behavior: Dict, network: Dict):
+        super().__init__()
+        self.static = static
+        self.behavior = behavior
+
+        self.global_features = capa.features.extractors.cape.global_.extract_features(self.static)
+
+
+    def extract_global_features(self) -> Iterator[Tuple[Feature, Address]]:
```

Comment:
code style CI action will complain about this formatting (too many newlines between methods), so please adjust to get those checks "green"